src/pages/Portfolio/Portfolio.js

```diff
@@ -1,11 +1,39 @@
 import React, { Component } from 'react';
+
+import { Modal } from 'components';
 import { Container, Subtitle } from './Portfolio.styles';
+import coinGecko from 'api/coinGecko';
 
 export class Portfolio extends Component {
+  state = {
+    isLoading: false,
+    coins: []
+  };
+  getCoins = async () => {
+    this.setState({ ...this.state, isLoading: true });
+    const { data } = await coinGecko.get(`/coins/markets`, {
+      params: {
+        vs_currency: 'usd',
+        order: 'market_cap_desc',
+        per_page: '250'
```

Comment:
use camelCase

Reply:
how can I use camel case for these params from the API ???